lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1260,7 +1268,7 @@ export class UserAgentApplication {
         this.registerCallback(serverAuthenticationRequest.state, scope, resolve, reject);
         this.logger.infoPii("Navigate to:" + urlNavigate);
         frameHandle.src = "about:blank";
-        this.loadIframeTimeout(urlNavigate, frameName, scope);
+        this.loadIframeTimeout(urlNavigate, frameName, scope).catch(error => reject(error));
```

Comment:
is there a reason why in some places we have `if (reject)` after catching the error and sometimes not?